src/index.html

```diff
@@ -6,9 +6,47 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html">
+        <img src="images/logo.png"
+        alt="MOYOLOGO" class="img">
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item" >
+            <a href="index.html" class="nav-link is-active">APPLE</a>
+          </li>
```

Comment:
![image](https://github.com/mate-academy/layout_moyo-header/assets/54936044/9b0adec0-9a95-4c3d-a91b-0282c1b50473)

Reply:
Still not fixed